proxy/src/main/java/org/dragonet/proxy/network/cache/EntityCache.java

```diff
@@ -67,6 +67,10 @@ public CachedEntity getClientEntity() {
 	}
 
 	public CachedEntity getByRemoteEID(long eid) {
+            if (!mapRemoteToClient.containsKey(eid))
```

Comment:
Either fix your formatting and brace style or don't commit at all

Reply:
file reformated